index.d.ts

```diff
@@ -613,72 +615,26 @@ type TemplateExpression =
 
 type Execa$<StdoutStderrType extends StdoutStderrAll = string> = {
 	/**
-	Same as `execa()` (including its return value) except both file and arguments are specified in a single tagged template string. For example, `` $`echo unicorns` `` is the same as `execa('echo', ['unicorns'])`.
-
-	It's important to note that quotes, backslashes, and spaces are automatically escaped and have no special meaning unless the `shell` option is used. This escaping behavior also applies to interpolated expressions such as strings (`` $`echo ${'string'}` ``), arrays of strings (`` $`echo ${['array', 'of strings']}` ``), and so on.
-
-	The `shell` option must be used if the `command` uses shell-specific features (for example, `&&` or `||`), as opposed to being a simple `file` followed by its `arguments`.
-
-	As a convenience, the result from previous `` $`command` `` or `` $.sync`command` `` calls can be used as template expressions in subsequent commands and `$`/`$.sync` will use the `stdout` value. See the example below `` with results from `$` or `$.sync` `` for more details.
-
-	@returns An `ExecaChildProcess` that is both:
-		- a `Promise` resolving or rejecting with a `childProcessResult`.
-		- a [`child_process` instance](https://nodejs.org/api/child_process.html#child_process_class_childprocess) with some additional methods and properties.
-	@throws A `childProcessResult` error
-
-	@example <caption>Basic</caption>
-	```
-	import {$} from 'execa';
-
-	const {stdout} = await $`echo unicorns`;
-	// const {stdout} = await $`echo ${'unicorns'}`;
-	// const {stdout} = await $`echo ${['unicorns', 'rainbows']}`;
+	Returns a new instance of `$` but with different default `options`. Consecutive calls are merged to previous ones.
```

Comment:
Those code comments were accidentally documenting `` $`command` `` instead of `$(options)`.